exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,7 @@
 // Declare your function here
+function createLongGreeting(a, b) {
+    return "I am " + a + " and I am " + b + " years old"
```

Comment:
As a general comment, how might you rewrite this as a template string? Usually, template strings are preferred to string concatenation (especially when there are many variables involved), as it's usually more readable.